server/bleep/src/webserver/studio.rs

```diff
@@ -177,14 +191,20 @@ pub async fn get(
 
     let context: Vec<ContextFile> =
         serde_json::from_str(&row.context).context("failed to deserialize context")?;
+    let doc_context: Vec<DocContextFile> = if let Some(c) = row.doc_context {
+        serde_json::from_str(&c).context("failed to deserialize doc context")?
+    } else {
+        Vec::new()
+    };
```

Comment:
If we migrate `doc_context` to be `NOT NULL`, we can remove special handling here.